src/matchers/toContainEntry/predicate.js

```diff
@@ -0,0 +1 @@
+export default (obj, [key, value]) => obj.hasOwnProperty && obj.hasOwnProperty(key) && obj[key] === value;
```

Comment:
I think this could be extended out to do a deep equals i.e. 

```js
import { equals } from 'expect/build/jasmine_utils';

export default (obj, [key, value]) => obj.hasOwnProperty && obj.hasOwnProperty(key) && equals(obj[key], value);
```

Reply:
Right, that sounds good, change should also be done in the other containEntry matchers to keep it consistent